providers/default.rb

```diff
@@ -107,25 +107,29 @@ def upgradeable?(name)
   end
 
   Chef::Log.debug("Checking to see if this chocolatey package is installed/upgradable: '#{name}'")
-  cmd = Mixlib::ShellOut.new("#{::ChocolateyHelpers.chocolatey_executable} version #{name} #{cmd_args}")
+  cmd = Mixlib::ShellOut.new("#{::ChocolateyHelpers.chocolatey_executable} upgrade -r --noop #{cmd_args} #{name}")
   cmd.run_command
-  !cmd.stdout.include?('Latest version installed')
+  result = cmd.stdout.chomp
+  return false if result.empty? # for chocolatey 0.9.9 + 0.9.9.1
```

Comment:
Can we please get this merged? This is a blocker on my workflow and I'd rather use the master branch if at all possible